sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -248,6 +249,18 @@ foreach (DocumentPage page in result.Pages)
     }
 }
 
+Console.WriteLine("Paragraphs:");
```

Comment:
I'm adding paragraph extraction to the Layout sample only. I believe other models have more specific features we should showcase in their samples.